CONTRIBUTING.md

```diff
@@ -26,6 +26,16 @@ Code for Fastify's **v1.x** is in [branch 1.x](https://github.com/fastify/fastif
 
 Declaring formal releases remains the prerogative of the lead maintainers. Do not bump version numbers in pull requests.
 
+## Plugins
+
+The contributors to the Fastify's plugins must attend the same rules of the Fastify repository with few adjustement:
+
+1. A release can be published by any member:
+  1. The SemVer version must be agreed with a lead maintainer if there is doubt.
```

Comment:
I don't know if this is really required. They should know SemVer and a PR should be never merged without a reviewer.

We should also use the version labels "major" .. to see how the maintainer it will handle.

Reply:
I think we can avoid this step. However we should add a line about Node version support for the fastify version, and the fact that we cannot drop old Node.js versions from plugins as well. Does it make sense?